### ISSUE TITLE
Plains biome in Y-256+

### ISSUE THREAD

**Issue by PuzzleChannel** [DeeChael/HigherNether#1]
![image](https://github.com/DeeChael/HigherNether/assets/129137837/896537d7-87f6-4160-829d-1a2ac9e5605f)

I have installed two mods: Amplified Nether and Higher Nether. As a result, the Plains biome is located above 256 heights. It's raining and ordinary monsters spawn.

**Comment by DeeChael** [comment]
Tested on Fabric 1.21 works fine
![image](https://github.com/DeeChael/HigherNether/assets/63186003/3f51c548-85f0-4455-b47f-769ffe30b44a)
Can you tell the exact version and modloader you using?

**Comment by DeeChael** [comment]
And is there any other mods installed, please list all the information that may useful

**Comment by PuzzleChannel** [comment]
![image](https://github.com/DeeChael/HigherNether/assets/129137837/034924aa-2425-4d96-96b1-7f2692e3b6a4)

In some areas, ill find regular biomes, but in other places, i got Plains.

**Comment by DeeChael** [comment]
World seed and the location plz, lemme check it

**Comment by PuzzleChannel** [comment]
seed -5682133114820172699
-83 273 279

**Comment by DeeChael** [comment]
wait r u sure the mod u installed is Higher Nether rather than Amplified Nether Height?
I didn't see Higher Nether in your mod list but Amplified Nether Height

**Comment by DeeChael** [comment]
well, we have the same datapack strucutre so it doesn't matter i think.
But I installed exactly same mods with your mod list but i cant reproduce your problem
![image](https://github.com/DeeChael/HigherNether/assets/63186003/a1285f0d-36cd-4c4a-8bb5-7eb234d7d12b)

**Comment by DeeChael** [comment]
also, with command locate cannot find the biome
![image](https://github.com/DeeChael/HigherNether/assets/63186003/f129f8d8-16e0-49d4-8683-0ac355d4e07c)

**Comment by Tanjiraaaaa** [comment]
> duplicated with #1

Yes I am aware of that, but I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod.
And there is no solution yet, so I figured I just make a new issue

**Comment by Tanjiraaaaa** [comment]
> duplicated with #1

Yes I am aware of that, but I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod.
And there is no solution yet, so I figured I just make a new issue

**Comment by Tanjiraaaaa** [comment]
I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod.
And there is no solution yet, so I figured I just make a new issue, my bad

**Comment by DeeChael** [comment]
> > duplicated with #1
> 
> Yes I am aware of that, but I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod. And there is no solution yet, so I figured I just make a new issue

it's same, but i cant reproduce it, it's weird and as this is just a datapack increase the building height i dont think it can cause this biome problem

**Comment by Tanjiraaaaa** [comment]
> > > duplicated with #1
> > 
> > 
> > Yes I am aware of that, but I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod. And there is no solution yet, so I figured I just make a new issue
> 
> it's same, but i cant reproduce it, it's weird and as this is just a datapack increase the building height i dont think it can cause this biome problem

Perhaps it's a problem with the Incendium mod?

**Comment by DeeChael** [comment]
> > > > duplicated with #1
> > > 
> > > 
> > > Yes I am aware of that, but I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod. And there is no solution yet, so I figured I just make a new issue
> > 
> > 
> > it's same, but i cant reproduce it, it's weird and as this is just a datapack increase the building height i dont think it can cause this biome problem
> 
> Perhaps it's a problem with the Incendium mod?

No i tested with Incendium but i cant reproduce, and the user open this issue didnt install incendium

**Comment by Tanjiraaaaa** [comment]
> > > > > duplicated with #1
> > > > 
> > > > 
> > > > Yes I am aware of that, but I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod. And there is no solution yet, so I figured I just make a new issue
> > > 
> > > 
> > > it's same, but i cant reproduce it, it's weird and as this is just a datapack increase the building height i dont think it can cause this biome problem
> > 
> > 
> > Perhaps it's a problem with the Incendium mod?
> 
> No i tested with Incendium but i cant reproduce, and the user open this issue didnt install incendium

What if I built what I needed to and then deleted the mod, do you think the build would stay?

**Comment by DeeChael** [comment]
> > > > > > duplicated with #1
> > > > > 
> > > > > 
> > > > > Yes I am aware of that, but I thought it'd be different since I'm using Incendium and the Amplified Nether Height mod. And there is no solution yet, so I figured I just make a new issue
> > > > 
> > > > 
> > > > it's same, but i cant reproduce it, it's weird and as this is just a datapack increase the building height i dont think it can cause this biome problem
> > > 
> > > 
> > > Perhaps it's a problem with the Incendium mod?
> > 
> > 
> > No i tested with Incendium but i cant reproduce, and the user open this issue didnt install incendium
> 
> What if I built what I needed to and then deleted the mod, do you think the build would stay?

it wont because if u removed the mod (datapack) the height limit is shrinked and the blocks above the height limit is illegal and it will be removed by the game

**Comment by DuckKin9** [comment]
![Image](https://github.com/user-attachments/assets/491b2d6c-7fc7-4f36-a157-95b7b875da39)
This my server's modlist.The version is 1.21fabric.I met the same problems at the location 105 323 -9 with the seed 20240624.
![Image](https://github.com/user-attachments/assets/e944ffe4-3360-4cca-8583-fc06043134f7)

**Comment by DuckKin9** [comment]
And there is not any more datapacks in the world except some recipes and enderman_holdablelist.
